routes/dashboard.js

```diff
@@ -70,6 +70,87 @@ router.get('/transfer', auth.isAuthenticated, function(req, res, next) {
     next(err);
   });
 });
+router.post('/transfer', function(req, res, next){
+  const transferAmount = req['body']['transferamount']; //Getting the amount from the user.
+  const transferFrom = req['body']['transferfrom']; //Getting the radio choice. 
+  const transferTo = req['body']['transferto']; //Getting the radio choice. 
+
+  console.log(transferAmount);
+
+  if(validAmount(transferAmount) == true){
+    transferMoney(transferAmount, transferFrom, transferTo, req.user._id);
+  }
+  else
+  {
+    req.flash('error', 'Amount is negative.');
+    return res.redirect('transfer');
+    // console.log('Amount is negative.');
+    // res.render('/transfer');
+  }
+  res.render('dashboard/transfer');
+})
+
+function transferMoney(temp_transfer_amount, temp_transfer_from, temp_transfer_to, user_ID){ //A function where the transferring will be happening. 
+  Account.find({user_ID: user_ID}, function(err, accounts){
+    if(err){
+      console.log(err);
+    }
+    console.log(accounts.length);
+    console.log(user_ID);
+    var tempAmountHold = 0;
+
+    if(temp_transfer_from == temp_transfer_to){
+      console.log("Can't transfer money in the same account."); //"Same Account" as in transferring from savings to savings, checkings to checkings, etc. 
```

Comment:
It would be better to redirect back to the transfer page and display the error to the user.
```suggestion
      req.flash('error', 'Can't transfer money in the same account.');
      return res.redirect('transfer');
```